reconcile/jenkins_job_builds_cleaner.py

```diff
@@ -36,11 +36,11 @@ def delete_builds(jenkins, builds_todel, dry_run=True):
 
 def get_last_build_ids(builds):
     builds_to_keep = []
-    if builds != []:
+    if builds:
         last_build = sorted(builds, key=lambda b: b["timestamp"], reverse=True)[0]
         builds_to_keep.append(last_build["id"])
 
-    for build in sorted(builds, key=lambda b: b["timestamp"], reverse=True):
+    for build in builds:
```

Comment:
But `builds` is not sorted anymore. I would suggest:

```python
def ...
    sorted_builds = sorted(builds, key=lambda b: b["timestamp"], reverse=True)
    if sorted_builds:
        last_build = sorted_builds[0]
    ...
    for build in sorted_builds:
    ....
```

Reply:
was not my best day, fixed